travel_platform/izi_travel_client.py

```diff
@@ -10,15 +10,18 @@ class IziTravelApiError(BaseExceptionHandler):
     pass
 
 
-class IziTravelClient(object):
+class BaseIziTravelApiClient(object):
 
-    def __init__(self):
-        self.api_key = "9ba66b8d-2bdc-4674-bdc4-3cc67e2c25af"
-        self.api_url = "https://api.izi.travel/"
+    API_URL = "https://api.izi.travel"
```

Comment:
Well, you may have something like `self.config` property of file created in your `__init__` method, then read appropriate variables from it, when it's needed.